### PR TITLE
Add backend support for sending custom acknowledgement messages

### PR DESCRIPTION
**Pull request by JeffersonBledsoe**
See collective/volto-form-block#57 for more details

### CODE REVIEW

**Comment by JeffersonBledsoe** [comment]
@cekk ready for review

**Comment by JeffersonBledsoe** [comment]
@cekk Happy to merge this?

File: src/collective/volto/formsupport/restapi/services/submit_form/post.py [hunk 1]
```diff
@@ -269,8 +277,30 @@ def send_data(self):
             if header_value:
                 msg[header] = header_value
 
+        should_send = self.block.get("send", [])
         self.manage_attachments(msg=msg)
-        self.send_mail(msg=msg, charset=charset)
+        if should_send:
```
Comment (cekk) [inline src/collective/volto/formsupport/restapi/services/submit_form/post.py:282]: Could be better to make this check before preparing the message? If not should_send you can return without doing anything..or not?

**Review by cekk — CHANGES_REQUESTED** [review]
What's the purpose of this pr? Send a message to the user that submitted the form?
Right now the email is only sent to the "owner" of the form, right (recipient field)?

LGTM but i think that you should do some additional things:

- update README to explain ne features
- update CHANGELOG (mentioning that this version need xxx version of the frontend)
-z (most important): upgrade-step that fix "send" field on already created blocks (from single string, to a list) or handle both use-case in the code to not break anything

**Comment by JeffersonBledsoe** [comment]
@cekk I've made all of the previously noted changes.

Unfortunately the frontend for this isn't merged or released yet. Would it be possible to have this PR merged so the functionality can be tested in the frontend, and then an additional PR made later before the next release to add documentation for the required frontend version for this feature?

**Review by tiberiuichim — APPROVED** [review]

**Pull request merged**